scripts/nats-bootstrap/nsc-bootstrap.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/sh
+
+set -e
+
+export NKEYS_PATH=/root/nsc/nkeys
+export NSC_HOME=/root/nsc/accounts
+
+if [ ! -f .nsc.env ]; then
+	echo '
+# NSC Environment Setup
+export NKEYS_PATH=/root/nsc/nkeys
+export NSC_HOME=/root/nsc/accounts
+' >nsc.env
+fi
+
+mkdir -p "$NKEYS_PATH"
+mkdir -p "$NSC_HOME"
+
+# create operator account
+nsc add operator --name KO
+nsc edit operator --service-url nats://nats:4222
+nsc edit operator --account-jwt-server-url nats://nats:4222
+
+# Create system account - required to interact with the nats server
+nsc add account --name SYS
+nsc add user --name sys
+
+# create servserservice Jetstream account - with permissions to list, create JS streams
+nsc add account --name serverservice
+nsc edit account --name serverservice --js-mem-storage -1 --js-consumer -1 --js-streams -1
+
+# generate a signing key to create a serverservice role
```

Comment:
Is there a secrets-store we could use for some of this? My concern is that making new keys on a deployment is probably fine for a sandbox environment, but won't be as nice in production.